brands/amenity/bank.json

```diff
@@ -3273,6 +3275,8 @@
     "tags": {
       "amenity": "bank",
       "brand": "National Bank",
+      "brand:wikidata": "Q634298",
+      "brand:wikipedia": "en:National Bank of Canada",
```

Comment:
This seems to be the wrong entry.
The canadian bank has been added with https://github.com/osmlab/name-suggestion-index/commit/42cba58c53de61cff05e937eeec2c41078638d00
This bank actually seems to be rather a generic (?) name when looking at Overpass Turbo: https://overpass-turbo.eu/s/Hf5 Though there are actually some in Canada.
Could you maybe say which one is more often used in Canada: "National Bank" or "Banque Nationale"?

Reply:
@bonbon12 Are you living in Canada and can answer my question? Or did you just researched it on the internet?

Reply:
I live in Canada, and its referred to as National Bank.